de-lib/src/main/java/org/iplantc/de/client/services/impl/AppUserServiceFacadeImpl.java

```diff
@@ -175,10 +175,7 @@ public void getAppDetails(final App app, DECallback<App> callback) {
         deServiceFacade.getServiceData(wrapper, new DECallbackConverter<String, App>(callback) {
             @Override
             protected App convertFrom(String object) {
-                Splittable split = StringQuoter.split(object);
-                AutoBean<App> appAutoBean = AutoBeanUtils.getAutoBean(app);
-                AutoBeanCodex.decodeInto(split, appAutoBean);
-                return appAutoBean.as();
+                return AutoBeanCodex.decode(factory, App.class, object).as();
```

Comment:
Really good catch!  I think I skimmed over this code too quickly and missed the intent.  What's strange is that if I remove this commit and the NPE fix commit, that doesn't seem to reintroduce the date parsing error, which is suspicious to me.  Either way, I'll go ahead and remove these two commits for now.